MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs

```diff
@@ -463,7 +473,15 @@ public override MvxBasePresentationAttribute CreatePresentationAttribute(Type vi
 
         public virtual Task<bool> CloseContentPage(IMvxViewModel viewModel, MvxContentPagePresentationAttribute attribute)
         {
-            return ClosePage(FormsApplication.MainPage, null, attribute);
+            var view = FindViewFromViewModel(viewModel);
```

Comment:
Please refactor this block of code so it's not repeated three times.

Reply:
I updated the code but it shows a failing test in `MvvmCross.UnitTest.Navigation.NavigationServiceTests.Test_NavigateCallbacks` but in my machine, it passes the test not sure why